oktaawscli/aws_auth.py

```diff
@@ -146,9 +153,46 @@ def write_sts_token(self, profile, access_key_id, secret_access_key, session_tok
 
         with open(self.creds_file, 'w+') as configfile:
             config.write(configfile)
-        print("Temporary credentials written to profile: %s" % profile)
+        sys.stderr.write("Temporary credentials written to profile: %s\n" % profile)
         self.logger.info("Invoke using: aws --profile %s <service> <command>" % profile)
 
+    def copy_to_default(self, profile):
+        """ Reads STS auth information from credentials file """
+        if not os.path.exists(self.creds_dir):
+            os.makedirs(self.creds_dir)
+        config = RawConfigParser()
+
+        if os.path.isfile(self.creds_file):
+            config.read(self.creds_file)
+
+        if not config.has_section(profile):
+            config.add_section(profile)
+
+        if config.has_option(profile, 'output'):
+            output = config.get(profile, 'output')
+
+        if config.has_option(profile, 'region'):
+            region = config.get(profile, 'region')
+
+        if config.has_option(profile, 'aws_access_key_id'):
+            access_key_id = config.get(profile, 'aws_access_key_id')
+
+        if config.has_option(profile, 'aws_secret_access_key'):
+            secret_access_key = config.get(profile, 'aws_secret_access_key')
+
+        if config.has_option(profile, 'aws_session_token'):
+            session_token = config.get(profile, 'aws_session_token')
```

Comment:
Do we need a temp variable here? Can this just be ` config.set('default', 'aws_security_token', config.get(profile, 'aws_session_token'))`

Reply:
Removed temporary variables and have config set inside of condition